README.md

```diff
@@ -1,17 +1,54 @@
 # Gatehouse
 
+This [Play Framework](https://www.playframework.com/) app provides an API for managing Identity accounts.
+The API forms a thin wrapper around our Identity Provider.
+
+While it's in a state of transition, it's also used to manage user service and marketing permissions.
+
+## Structure
+
+The main components of the app are:
+
+```mermaid
+classDiagram
+    class UserController
+    class UserService
+    class legacyIdentityDbUserService
+    class OktaUserService
+    <<NotCreatedYet>> OktaUserService
+    class CompositeUserService
+    <<NotCreatedYet>> CompositeUserService
+    UserController <--> UserService: User
+    UserService <|-- legacyIdentityDbUserService
+    UserService <|-- CompositeUserService
+    UserService <|-- OktaUserService
+    UserService: healthCheck()
+    UserService: fetchUserByIdentityId()
+    UserService: ...()
+    CompositeUserService --* legacyIdentityDbUserService
+    CompositeUserService --* OktaUserService
+```
+
+## Running the app locally
+
+1. Set up a [local PostgreSQL service](https://github.com/guardian/identity-platform/tree/main/postgres) listening on port 5555.
+2. Install local AWS credentials for the Identity account.
+3. Run the app with `sbt run`.
```

Comment:
maybe here is a good way to mention about the required env variable to set the aws profile, unless you are planning to refactor so that it's not needed anymore of course

Reply:
I'll add it here for this PR and do the refactor separately

Reply:
Added a step to instructions in bb49f6dcea25cc6e2641771dc0d367955b2a2875